docs/install/prerequisites.rst

```diff
@@ -192,35 +192,108 @@ To install for the currently active kernel run the command corresponding to your
 
 .. _group_permissions:
 
-Setting permissions for groups
+Configuring permissions for GPU access
 ================================================================
 
-This section provides steps to add any current user to a video group to access GPU resources. We
-recommend using the video group for all ROCm-supported operating systems.
+There are two primary methods to configure GPU access for ROCm: group membership or
+udev rules. Each method has its own advantages, and the choice depends on your 
+specific requirements and system management preferences.
+
+Using Group Membership
+--------------------------------------------------------------------
+
+By default, GPU access is managed through membership in the ``video`` and ``render`` groups.
+The ``video`` and ``render`` groups are system groups in Linux used to manage access 
+to graphics hardware and related functionality. Traditionally, the ``video`` group is used 
+to control access to video devices, including graphics cards and video capture devices. 
+The ``render`` group is more recent and specifically controls access to GPU rendering capabilities 
+through Direct Rendering Manager (DRM) render nodes.
 
 1. To check the groups in your system, issue the following command:
 
    .. code-block:: shell
 
        groups
 
-2. Add yourself to the ``render`` and ``video`` group using the command:
+2. Add yourself to the ``video`` and ``render`` groups:
+
+   .. code-block:: shell
+
+      sudo usermod -a -G video,render $LOGNAME
+
+3. Optionally, add other users to the ``video`` and ``render`` groups:
+
+   .. code-block:: shell
+
+      sudo usermod -a -G video,render user1
+      sudo usermod -a -G video,render user2
+
+4. To add all future users to the render and video groups by default, run the following commands:
+
+   .. code-block:: shell
+
+      echo 'ADD_EXTRA_GROUPS=1' | sudo tee -a /etc/adduser.conf
+      echo 'EXTRA_GROUPS=video' | sudo tee -a /etc/adduser.conf
+      echo 'EXTRA_GROUPS=render' | sudo tee -a /etc/adduser.conf
+
+Using udev Rules
+--------------------------------------------------------------------
+Udev rules offer a flexible way to manage device permissions. They apply system-wide, can be 
```

Comment:
Good suggestion. I like the idea of rewording the sentence to avoid capitalizing udev. I think the actual utility is udevadm, so we technically don't want it in quotes. We could also potentially use the imperative voice here, for example, "Manage device permissions in a flexible manner using udev rules". @AMD-melliott feel free to rework this according to your preference.

Reply:
lgtm